src/index.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+import { Provider } from 'react-redux';
+import setupStore from './redux/store';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+
+const store = setupStore();
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
-  </React.StrictMode>
+    <Provider store={store}>
+      <App />
+    </Provider>
```

Comment:
- [ ] It would be better to display the greeting messages in the browser, however, we are not seeing the messages in the browser when we start the app, although we have stored data in the database, as we can see in the screenshot below
![Screenshot from 2023-07-11 09-03-08](https://github.com/zeff96/hello_react_front_end/assets/46388113/e9992611-857f-468a-8088-1ac03378be0b)